deployment_and_ssl_doc.md

```diff
@@ -107,10 +154,13 @@ word `exit`.
 
 ## Create an SSL Certificate
 
-For this step you will need to fork the Repo that can be found at this url
+Before creating an SSL certificate you must have a domain set up that you want
+to use.
+
+For this step you will need to fork the repo that can be found at this url
```

Comment:
Can we please make a comment that it would be great to get a chance to test the *regular* certbot nginx method at the next install:

```
sudo certbot --nginx certonly
```

see `advanced` panel at https://certbot.eff.org/#ubuntuxenial-nginx  This method would remove all of the following steps down to *Use Edeliver to Deploy Application to Azure* (although please continue to note the instructions on how to enter/use the actual certbot SSL certificate generation screens).